scripts/extract.py

```diff
@@ -91,11 +91,6 @@ def main(args):
             if torch.cuda.is_available() and not args.nogpu:
                 toks = toks.to(device="cuda", non_blocking=True)
 
-            # The model is trained on truncated sequences and passing longer ones in at
-            # infernce will cause an error. See https://github.com/facebookresearch/esm/issues/21
-            if args.truncate:
```

Comment:
Now argparser and L76 will need to be changed to include `truncation_seq_length`, you can set a default value of 1022 in argparser.

Reply:
Should I remove the `--truncate` parser argument, since now the truncation is fully handled by `truncation_seq_length`?

Reply:
yes removing is good. But I'd want to preserve the default behavior of this script, ie add to the argparser the flag `--truncation-seq-length` and set its default to 1022

Reply:
Done!

Reply:
Great!